shell/src/main/java/alluxio/shell/command/RmCommand.java

```diff
@@ -65,21 +64,26 @@ protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioExcepti
           path.getPath() + " is a directory, to remove it, please use \"rm -R <path>\"");
     }
 
-    DeleteOptions options = DeleteOptions.defaults().setRecursive(recursive);
-    if (cl.hasOption(REMOVE_UNCHECKED_OPTION_CHAR)) {
-      options.setUnchecked(true);
-    }
+    boolean isAlluxioOnly = cl.hasOption("alluxioOnly");
+    //System.out.println("isAlluxioOnly: " + isAlluxioOnly);
+    DeleteOptions options = DeleteOptions.defaults()
+        .setRecursive(recursive).setAlluxioOnly(isAlluxioOnly);
     mFileSystem.delete(path, options);
-    System.out.println(path + " has been removed");
+    if (!isAlluxioOnly) {
+      System.out.println(path + " has been removed");
+    } else {
+      System.out.println(path + " just been removed in Alluxio Space, but Ufs space still exists");
```

Comment:
path + " has been removed from Alluxio space, but still exists in the under storage."